src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -112,6 +112,28 @@ public without sharing class UTIL_OrgTelemetry_SVC {
         Database.executeBatch(new UTIL_OrgTelemetry_BATCH(), 1);
     }
 
+    /**
+     * @description Determines if we should execute the telemetry code based on if the created custom metdata types
+     * are NPSP metadata type objects. 
+     * @param customMetadataTypeSet
+     */
+    @Future
+    public static void asyncProcessCMTChange(Set<String> customMetadataTypeSet) {
+
+        // We need to parse and check so that only NPSP specific actions will have telemetry execution.
+        // We don't want this to run for Abacus. 
```

Comment:
What is the comment about not wanting to run for Abacus referencing?

Reply:
Updated the comment to be more clear